client/js/components/IssueRelationships.tsx

```diff
@@ -67,22 +62,40 @@ export class IssueRelationships extends React.Component<Props, States> {
     super(props);
 
     this.state = {
-      relationships: props.relationships,
+      relationships: props.issueData.issue.relationships,
       reqRelTyp: RelationshipTypeEnum.RELATED_TO,
-      reqRelDestIds: ''
+      reqRelDestIds: '',
+      canResolve: true,
     };
-    this.Service = new IssueService(props.issueId);
+    this.Service = new IssueService(props.issueData.issue.id);
+  }
+
+  componentDidUpdate() {
+    /*this.setState({
+      canResolve: this.canResolveBug()
+    });*/
+  }
+
+  canResolveBug() {
+    const { relationships } = this.state;
+
+    for (const relationship of relationships) {
+      if (relationship.type.id === this.getConfigValue('default_bug_dependant') &&
```

Comment:
@waltergar We should get the config options once before the loop and have them in variables that are used within the loop.